src/styles/layout.scss

```diff
@@ -571,11 +571,17 @@ h6:hover .autolink-headers {
 
 .blog-grid-container {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+  grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
```

Comment:
This is too big mobiles so it produces horizontal scroll

Reply:
I am going to add an override on the `mobile.scss` file.